intel_extension_for_transformers/transformers/__init__.py

```diff
@@ -44,7 +44,11 @@
     MixedPrecisionConfig,
     BitsAndBytesConfig,
     SmoothQuantConfig,
-    WeightOnlyQuantConfig,
+    RtnConfig,
+    AwqConfig,
+    TeqConfig,
+    GPTQConfig,
```

Comment:
Why we use different config ？
I HF format, all method share one "quantization_config"
and use "quant_method": "gptq" to distinguish the methods

Reply:
all different have the save_pretrained method to save quantizate_config to config.json file, use different config will write different quant_method to config.json and quantize_config.json, it is the same behavoir with transformers.